Core/Utilities/Subprocess.py

```diff
@@ -368,6 +368,9 @@ def __readFromFile( self, fd, baseLength, doAll = True ):
         if nB == "":
           break
         dataString += nB
+        if len( dataString ) > 10000000:
```

Comment:
I gave the number a name, but I didn't make it global, because it is not used anywhere else.
Added comments, though there is also the commit message and the error message...